spec/models/proposal_spec.rb

```diff
@@ -81,4 +81,144 @@
       expect(proposal.users).to eq([proposal.requester])
     end
   end
+
+  describe '#approvers=' do
+    it 'sets initial approvers' do
+      proposal = FactoryGirl.create(:proposal)
+      approvers = [FactoryGirl.create(:user), FactoryGirl.create(:user), FactoryGirl.create(:user)]
+      proposal.approvers = approvers
+      expect(proposal.approvals.count).to be 3
+      expect(proposal.approvers).to eq approvers
+    end
```

Comment:
Minor style thing: for specs that are longer than ~3 lines, I like to put whitespace between the "setup", "action", and "assertion" sections. In other words,

```ruby
proposal = FactoryGirl.create(:proposal)
approvers = [FactoryGirl.create(:user), FactoryGirl.create(:user), FactoryGirl.create(:user)]

proposal.approvers = approvers

expect(proposal.approvals.count).to be 3
expect(proposal.approvers).to eq approvers
```